src/bruno/BowlingGame.test.ts

```diff
@@ -0,0 +1,68 @@
+import BowlingGame from "./BowlingGame";
+
+const rollMany = (n: number, pins: number) => {
+  for (let index = 0; index < n; index++) {
+    bowlingGame.roll(pins);
+  }
+};
+
+const rollSpare = () => {
+  bowlingGame.roll(5);
+  bowlingGame.roll(5);
+};
+
+const rollStrike = () => {
+  bowlingGame.roll(10);
+};
+
+let bowlingGame: BowlingGame;
+beforeEach(() => {
+  // Arrange
+  bowlingGame = new BowlingGame();
+});
+
+describe("BowlingGame", () => {
+  it("should test a gutter game", () => {
+    // Act
+    rollMany(20, 0);
+    // Assert
+    expect(bowlingGame.score()).toEqual(0);
+  });
+
+  it("should test an all ones game", () => {
+    // Act
+    rollMany(20, 1);
+
+    // Assert
+    expect(bowlingGame.score()).toEqual(20);
+  });
+
+  it("should test one spare game", () => {
+    // Act
+    rollSpare();
```

Comment:
Seque esto está propuesto en la solución original, pero para mi sigue siento más claro (y creo que correcto), lo siguiente:

```suggestion
    bowlingGame.roll(5);
    bowlingGame.roll(5); // Spare
```